Sources/APPX.cpp

```diff
@@ -102,6 +104,15 @@ namespace appx {
             for (const auto &fileNamePair : fileNames) {
                 const std::string &archiveName = fileNamePair.first;
                 const std::string &fileName = fileNamePair.second;
+
+                const std::string suffix = "AppxBundleManifest.xml";
+                if (bundle &&
+                        suffix.size() < archiveName.size() &&
+                        std::equal(suffix.rbegin(), suffix.rend(), archiveName.rbegin())) {
```

Comment:
Why not specify the AppxBundleManifest.xml file as the value for the `-b` option? See how we handle the signing key.